taxonomies/categories.txt

```diff
@@ -98310,74 +98406,1356 @@ en:miscellaneous pet food, miscellaneous animal food
+ciqual_food_code:en:31026
+ciqual_food_name:en:White chocolate bar, with dried fruits (nuts, almonds, raisins, praline)
+ciqual_food_name:fr:Chocolat blanc aux fruits secs (noisettes, amandes, raisins, praliné) , tablette
+
+<en:Stuffed wafers
+en:Stuffed fruits wafers
+fr:Gaufrettes fourrées aux fruits
+hu:Töltött gyümölcs ostyák
+nl:Vruchtenwafeltjes
+
+<en:Viennoiseries
+en:Brioche breads
+fr:Pains briochés
+nl:Briochebrood
+
+<fr:Fromages blancs
+fr:Fromages blancs aromatisés
+
+<en:Cakes
+en:Almond cakes
+de:Mandelkuchen
+fr:Gâteaux aux amandes
+nl:Amandelcake
+ciqual_food_code:en:24665
+ciqual_food_name:en:Almond cake
+
+<en:Cakes
+en:Soft cake with nuts
+fr:Gâteau moelleux aux fruits à coque
+ciqual_food_code:en:23938
+ciqual_food_name:en:Soft cake with nuts
+ciqual_food_name:fr:Gâteau moelleux aux fruits à coque
+
+<en:Cakes
+en:Soft cake filled with fruits, Mini sponge roll filled with fruits
+fr:Gâteau moelleux fourré aux fruits, Mini-roulé fourré aux fruits, Mini-cake fourré aux fruits
+ciqual_food_code:en:23941
+ciqual_food_name:en:Soft cake filled with fruits, mini sponge roll type
+ciqual_food_name:fr:Gâteau moelleux fourré aux fruits type mini-roulé ou mini-cake fourré
+
+<en:Sweet mousses
+en:Fruit mousses, Refrigerated fruit mousse
+fr: Mousses aux fruits, Mousse aux fruits rayon frais
+nl:Vruchtenmousse
+ciqual_food_code:en:39228
+ciqual_food_name:en:Fruit mousse, refrigerated
+ciqual_food_name:fr:Mousse aux fruits, rayon frais
+
+<en:Desserts
+en:Vacherins
+fr:Vacherins
+nl:Vacherin
+
+<en:Pastries
+en:Strudels
+de:Strudel
+fr:Strudels
+nl:Strudel
+
+<en:Pastries
+<en:Desserts
+en:Charlottes
+fr:Charlottes
+nl:Charlotte
+
+<en:Charlottes
+en:Chocolate Charlottes
+fr:Charlottes au chocolat
+nl:Chocola Charlottes
+
+<en:Charlottes
+en:Fruit Charlottes
+fr:Charlottes aux fruits
+nl:Vruchtencharlotte
+ciqual_food_code:en:23531
+ciqual_food_name:en:Fruit charlotte
+ciqual_food_name:fr:Charlotte aux fruits
+
+<en:Pastries
+<en:Desserts
+en:Crumbles
+fr:Crumbles
+nl:Kruimelschotels, Kruimelschotel
+wikidata:en:Q2006073
+
+<en:Crumbles
+en:Apple crumbles
+fr:Crumbles aux pommes
+ciqual_food_code:en:23493
+ciqual_food_name:en:Apple crumble
+ciqual_food_name:fr:Crumble aux pommes
+
+<en:Terrines
+en:Fish terrines
+fi:kalaterriinit
+fr:Terrines de poisson
+nl:Visterrines
+pnns_group_2:en:Fish and seafood
+
+<en:Cheeses
+en:Mexican cheeses, cheeses from Mexico
+de:Mexikanischer Käse
+es:Quesos mexicanos, Quesos de México
+fi:meksikolaiset juustot
+fr:Fromages mexicains, Fromages du Mexique
+
+<en:Mexican cheeses
+en:Panela cheeses
+es:Quesos panelas
+fr:Fromages panelas
+
+<en:Salts
+en:Liquid salts
+es:Sales liquidas, sal liquida
+fr:Sels liquides
+
+en:skewers, Kebabs, kebab, kabobs, kabob, kababs, kabab
+fi:vartaat
+fr:Brochettes, brochette
+nl:Spiesjes
+
+en:edible tableware
+de:essbares Geschirr
+fr:Couverts comestibles, Vaisselles comestibles
 
-<en:Proposed for deletion
-en:Empty, Actually empty, No data
-ru:Пустое, Пустые
```

Comment:
I don't think that's right

Reply:
It is not very relevant as it is not related to the product. Wehave the states to track completeness.